man/org.freedesktop.systemd1.xml

```diff
@@ -8026,6 +8046,8 @@ node /org/freedesktop/systemd1/unit/dev_2dsda3_2eswap {
 
     <!--property RootVerity is not documented!-->
 
+    <!--property RootExtensions is not documented!-->
```

Comment:
please document the new property...

Reply:
mmh I'm not entirely sure if we want to document these kind of properties explicitly? I mean, it's a 1:1 mapping to the unit setting after all - name and values coincide

Reply:
Then please mention it in the text. Even something as simple as: "The following properties map 1:1 to corresponding settings in the unit file: RootVerity, RootExtension, ..., see systemd.service(5) for their meaning."

Reply:
Ok, done